java/src/jmri/jmrix/mqtt/Bundle.properties

```diff
@@ -58,3 +58,4 @@ MessageLastWill=lost
 MqttMast = MQTT Signal Mast
 AllowUnLitLabel                      = This Mast can be unlit
 DisableAspectsLabel                  = Disable specific Aspects
+MakeLabel = Topic Suffix:
```

Comment:
I will approve this PR but I think it would be better if the property was named `TopicSuffix`. The property `MakeLabel` is anonymous and doesn't give much clue what it's about.

The property MakeLabel comes from jmri.NamedBeanBundle where it is:
```
# use the MakeLabel key to add a localized interpunction after the above (or any other) item
# called in code as Bundle.getMessage("MakeLabel", Bundle.getMessage("BeanName")) so translation must include {0}
MakeLabel      = {0}:
```

Reply:
Aha!  Thanks for figuring that out.

And that explains why there are places in the interface where we get :: sequences:  A label has a :, and then it's run through MakeLabel to add a second : character.  Most of the time it would be better to have the "localized inter punctuation" in the actually translated term, I think.  But there are over a hundred uses of this, so that's a project for another day.

Will fix up this PR.

Reply:
When it comes to finding properties like this, I search thru the files `*Bundle.properties`. That way I search only the English property files.